.github/workflows/build-container.yml

```
@@ -52,6 +52,17 @@ jobs:
         tag_list=$(IFS=,; echo "${tags[*]}")
         echo "::set-output name=taglist::${tag_list}"
 
+    - id: generate-image-base
+      env:
+        DVER: ${{ matrix.dver}}
+      run: |
+        case "$DVER" in
+          8) image_base="quay.io/centos/centos:stream8";;
```

```suggestion
          9) image_base="quay.io/centos/centos:stream9";;
          8) image_base="quay.io/centos/centos:stream8";;
```
might as well do this while we're at it.

---

Yeah, I thought about it but decided against it since we don't have any EL9 OSG repos so the build will fail anyway. I don't feel too strongly about it so let's just do it